src/main/java/org/candlepin/hostedtest/HostedTestCloudRegistrationAdapter.java

```diff
@@ -222,4 +223,13 @@ private boolean isOwnerEntitledToProducts(String ownerKey, Collection<String> pr
         return true;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String checkCloudAccountOrgIsReady(String cloudAccountID, CloudProvider cloudProviderShortName,
+        String cloudOfferingID)
+        throws OrgForCloudAccountNotCreatedYetException, CouldNotEntitleOrganizationException {
```

Comment:
Need to update `CouldNotEntitleOrganizationException` to `OrgForCloudAccountNotEntitledYetException`

Reply:
done